packages/lit-dev-content/site/tutorials/content/async-directive/05.md

```diff
@@ -0,0 +1,77 @@
+To make our directive update asynchronously, we'll need to start a timer that
+re-renders the time elapsed.
+
+Add a function that starts a periodic 1s timer if it's not already started.
+Since directives are stateful, we can use a class field to store the timer's
+handle, and use that to start the timer only once:
+
+{% switchable-sample %}
+
+```ts
+  timer: number | undefined;
+
+  ensureTimerStarted() {
+    if (this.timer === undefined) {
+      this.timer = setInterval(() => {
+        /* do some periodic work */
+      }, 1000);
+    }
+  }
+```
+
+```js
+  timer = undefined;
+
+  ensureTimerStarted() {
+    if (this.timer === undefined) {
+      this.timer = setInterval(() => {
+        /* do some periodic work */
+      }, 1000);
+    }
+  }
+```
+
+{% endswitchable-sample %}
+
+
+We will start the timer in the `update` lifecycle callback.  The default
+implementation of `update` simply calls `render`, but since `update` is [_not_
+called during
+SSR](https://lit.dev/docs/templates/custom-directives/#differences-between-update()-and-render()),
+`update` is the right spot to start/subscribe to asynchronous tasks we don't
+want running on the server.
+
+Add the `update` callback. Call `ensureTimerStarted()` and then return the
+result of `render`. Note that we only want the timer running if the directive is
```

Comment:
Why not call `super.update`?

Reply:
In terms of understanding what's going on, `return super.update(part, [time])` just seems fussier and more obscure than `return this.render(time)`. 🤷